### PR TITLE
added dns table docs in readme

### PR DESCRIPTION
**Pull request by mik-tf**
# Work Done

- Added a table to summarize clearly how to set the records.
- Added info on each network file .secrets.env-example that the wallet needs to be funded on the chain itself

### CODE REVIEW

File: docker-compose/readme.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ To start a full Grid backend stack one needs the following:
 - min of 32GB RAM
 - min of 1TB SSD storage (high preference for NVMe based storage)
 
+#### Setting the DNS Records
+
+The following table explicitly shows how to set the A and AAAA records for your domain.
+
+| Type | Host | Value          |
+| ---- | ---- | -------------- |
+| A    | \*   | <ipv4_address> |
```
Comment (coesensbert) [inline docker-compose/readme.md:29]: I'm not sure you need the \ in front of the *. We use name.com and we only need to add *

**Review by coesensbert — APPROVED** [review]
:+1:

File: docker-compose/readme.md (hunk 1 above)
Comment (mik-tf) [reply docker-compose/readme.md:29]: Yes indeed it's because the file is in markdown. It's to be sure that we see the symbol.

Comment (mik-tf) [reply docker-compose/readme.md:29]: image

**Pull request merged**